addons/hr/res_config.py

```diff
@@ -48,8 +48,8 @@ class hr_config_settings(osv.osv_memory):
             help ="""This installs the module account_analytic_analysis, which will install sales management too."""),
         'module_hr_payroll': fields.boolean('Manage payroll',
             help ="""This installs the module hr_payroll."""),
-        'module_website_hr_recruitment': fields.boolean('Publish jobs on your website',
-            help ="""This installs the module website_hr_recruitment"""),
+        'module_hr_material': fields.boolean('Manage several material categories like IT, Office material, Engine, ...',
```

Comment:
We do not propose to install modules in setting anymore; setting are there to configure things or choose options. So please remove this.